templates/juniper_base_client.py

```diff
@@ -71,9 +71,9 @@ def __init__(
 
         ws_protocol = "wss" if use_tls else "ws"
         self.ws_url = f"{ws_protocol}://{host}:{port}/subscriptions/graphql"
-        self.ws_headers = {}
+        self.ws_headers: dict = {}
```

Comment:
@andrew-lee-architect does this field even do anything? there's no instance of it being added to